### ISSUE TITLE
Building errors

### ISSUE THREAD

**Issue by Vinceti** [DigitalRune/DigitalRune#2]
Hi,
I have managed to compile the projects to create the dlls to replace the older ones in my interop-UWP application. But now I need to create the following 4 dlls for the Pipeline:

DigitalRune.Animation.Content.Pipeline.dll
DigitalRune.Geometry.Content.Pipeline.dll
DigitalRune.Graphics.Content.Pipeline.dll
DigitalRune.Mathematics.Content.Pipeline.dll

But I can't manage to do that because I get some errors. Has anyone created this files succesfully? it could be possible for anyone to upload them?
Thanks in advance.

**Comment by LtKain** [comment]
I'll try this weekend to compile everything .. i am using the an old version with the license :)
Do you know what is  "DigitalRune-Editor" ? It wasn't in the licensed version so i'am wondering if it is a 3d editor to integrate object in the engine.

**Comment by Vinceti** [comment]
I was using a trial version which expires today, so I must say goodbye temporarily to my app :-(
I haven't read so much about the Editor, but it probably needs the same libraries as the pipeline tool to build the content. The fact we didn't hear anything about this tool until this open release makes me guess it is incomplete.
I thought I just needed those "content.pipeline" files, but actually I think I haven't created correctly some of the other neither. So I hope that anyone is able to compile all them correctly.
Thanks for your response, I will be awaiting your progress with the compilation ;-)

**Comment by luxbinary** [comment]
This project is really not the easiest thing to build but here is how it worked for my for the Windows build:
- Execute the Protobuild.exe at the Monogame folder
- At the Monogame solution replace the error parts with the original code from the monogame project. You need to create an instance for PVRTexture. Should be at Etc1BitmapContent.cs, PvrtcBitmapContent.cs and DxtBitmapContent.cs
- Build Monogame as Release
- Restore Nuget Packages at DigitalRune-MonoGame-Windows.sln
- Build DigitalRune.csproj as Release
- Build DigitalRune.CommandLine as Release (C#6 features!!! nameof)
- Build Tools/Pack as Release
- Build-Content-Release.cmd
- Build-Content-MonoGame-Windows.cmd
- Build the Samples

Hope this helps

**Comment by Vinceti** [comment]
Finally got it! I have succesfuly built the Universal Windows references for muy Interop applications.

I downloaded "MonoGame.Dependencies" from GitHub so I could repair somo missing references for MonoGame:
https://github.com/MonoGame/MonoGame.Dependencies

After that, just recompiled all the libraries in cascade, following the Dependency graph so I could determine the proper compilation order:
https://www.digitalrune.com/Documentation/media/Dependency-Graph.png

Finally I replaced the old references with these new other, and re-added the content file. For some reason, the path of the content file reset to the sample content file I used at the very beginning of the project; I almost went crazy until I realized it :-S

Hope it helps other people trying to build Universal Windows interop applications. Thanks you so much guys for your help.
